### ISSUE TITLE
Rubymine - mirrord doesn't load

### ISSUE THREAD

**Issue by aviramha** [metalbear-co/mirrord-vscode#60]
ruby-3.1.4

**Comment by infiniteregrets** [comment]
This is really just a extension-level issue from my research, I had the Ruby extension installed by Peng Lv and the debug session wont start even without mirrord, failing with `"Debugger terminal error: Process failed: spawn rdebug-ide ENOENT" `, on looking at some open issues i found this https://github.com/rubyide/vscode-ruby/issues/214. Had to add 
```
"env": {
  "PATH": "/Users/mehularora/.rvm/gems/ruby-3.1.4/bin:/Users/mehularora/.rvm/gems/ruby-3.1.4@global/bin:/Users/mehularora/.rvm/rubies/ruby-3.1.4/bin:/Users/mehularora/.rvm/bin:/Users/mehularora/.nvm/versions/node/v18.17.1/bin:/Users/mehularora/.local/bin:/opt/homebrew/opt/ruby/bin:/Users/mehularora/.krew/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/VMware Fusion Tech Preview.app/Contents/Public:/usr/local/share/dotnet:~/.dotnet/tools:/Applications/Wireshark.app/Contents/MacOS:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/Users/mehularora/.cargo/bin:/Users/mehularora/Library/Application Support/JetBrains/Toolbox/scripts:/usr/local/bin:/usr/local/share/dotnet",
  "GEM_HOME": "/Users/mehularora/.rvm/gems/ruby-3.1.4",
  "GEM_PATH": "/Users/mehularora/.rvm/gems/ruby-3.1.4:/Users/mehularora/.rvm/gems/ruby-3.1.4@global",
  "RUBY_VERSION": "ruby-3.1.4"
}
```
make the session even start without mirrord in the first place, but no breakpoints hit, on debugging with the vscode extension our flow is correct, we fetch the correct binary to load into, its the extension thats not maintained and has been archived. Now on switching to shopify's extension https://github.com/Shopify/vscode-shopify-ruby everything works perfectly, with/without mirrord and no env interference takes place too, but i will add the abovementioned env for https://github.com/metalbear-co/mirrord/issues/1892 as a precaution.

tldr, `vscode-ruby` doesnt work even without mirrord and using the shopify extension works
